style.css

```diff
@@ -0,0 +1,57 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:is(h1, h2, h3, p) { margin: 15px 0; }
+
+a { text-decoration: none; }
+
+body {
+  font: 400 15px/24px Poppins, sans-serif;
+  color: #344563;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  padding: 0.7% 3.4%;
+  width: 100%;
+  left: 0;
+  top: 0;
+  position: fixed;
+  background-color: #fff;
+  justify-content: space-between;
+}
```

Comment:
- [ ] The `header/navbar` section overrides the background image. As per the given template, there should be some space between `header` and `top of the background image`. Kindly fix this issue.
  - ***Given Design***
  ![image](https://user-images.githubusercontent.com/60083437/219151949-8392f9d7-dafa-40fa-b2a8-774b0138eb54.png)
  - ***Your Design***
  ![image](https://user-images.githubusercontent.com/60083437/219152260-3c8b1fe6-27a4-41a5-9060-d479458a2cd7.png)